neo/rawio/openephysrawio.py

```diff
@@ -255,14 +256,15 @@ def _parse_header(self):
         # Annotate some objects from coninuous files
         self._generate_minimal_annotations()
         bl_ann = self.raw_annotations['blocks'][0]
-        for seg_index in range(nb_segment):
-            seg_ann = bl_ann['segments'][seg_index]
+        for seg_index, oe_index in enumerate(oe_indices):
+            seg_ann = bl_ann['segments'][oe_index]
```

Comment:
No here it is seg_index, because neo.io route annoatations with this seg_index

Reply:
Oh yes absolutely, my mistake.